src/pages/SpacesDashboard/SpacesDashboard.scss

```diff
@@ -26,15 +26,15 @@
   &__cards {
     display: grid;
     width: 100%;
-    grid-template-columns: repeat(4, 1fr);
+    grid-template-columns: repeat(3, 1fr);
     grid-template-rows: max-content;
     grid-gap: font-size--root(1);
 
-    @media (max-width: $media-breakpoint--lg) {
-      grid-template-columns: repeat(4, 1fr);
+    @media (max-width: $media-breakpoint--xxl) {
+      grid-template-columns: repeat(2, 1fr);
     }
 
-    @media (max-width: $media-breakpoint--md) {
+    @media (max-width: $media-breakpoint--xl) {
       grid-template-columns: repeat(1, 1fr);
     }
```

Comment:
🚀  Noice. And what happens on wider screens? Do cards expand or does just the space between them?

Reply:
refer to https://github.com/sparkletown/sparkle/pull/2598#discussion_r757473693

cards resize just fine while spaces stay the same, I've tried to play around with different resolutions and didn't find any issues with cards so far